pkg/chunked/storage_linux.go

```diff
@@ -164,26 +164,6 @@ func copyFileContent(srcFd int, destFile string, dirfd int, mode os.FileMode, us
 	return dstFile, st.Size(), nil
 }
 
-// GetTOCDigest returns the digest of the TOC as recorded in the annotations.
-// This is an experimental feature and may be changed/removed in the future.
-func GetTOCDigest(annotations map[string]string) (*digest.Digest, error) {
```

Comment:
This is an API break. c/storage isn’t quite as strict about that as c/image, but still, a forwarding stub here would not be hard to add.